### PR TITLE
[stable/mariadb] add settings that can be overridden for Discretionary Access Controls

### PR DESCRIPTION
**Pull request by PaulCharlton**
add settings that can be overridden for Discretionary Access Controls - runAs, fsGroup.
ref : https://kubernetes.io/docs/tasks/configure-pod-container/security-context/

### CODE REVIEW

**Comment by PaulCharlton** [comment]
e-signed the CLA

**Comment by foxish** [comment]
LGTM, but looks like there are conflicts now. Sorry @PaulCharlton, can you rebase?

File: stable/mariadb/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: mariadb
-version: 2.1.3
+version: 2.1.4
```
Comment (juan131) [inline stable/mariadb/Chart.yaml]: Current version is 2.1.10
Please rebase from master and bump the version to 2.1.11

**Review by juan131 — COMMENTED** [review]
LGTM! Please fix the version on Chart.yml

**Comment by prydonius** [comment]
Tests are failing:
```
I0319 11:23:50.586] #   `mysql -h mariadb-6646 -uroot -p$MARIADB_ROOT_PASSWORD -e 'show databases;'' failed
I0319 11:23:50.586] # ERROR 2005 (HY000): Unknown MySQL server host 'mariadb-6646' (-2)
```

could you look into it?

**Comment by PaulCharlton** [comment]
taking a look

**Comment by PaulCharlton** [comment]
DNS resolution failed during script run.   Unknown cause.  I would try re-running CI first to see if it is a transient K8S failure, or if it is some sort of race condition in the propagation of DNS name resolution.

```
I0319 11:23:50.586] #   `mysql -h mariadb-6646 -uroot -p$MARIADB_ROOT_PASSWORD -e 'show databases;'' failed
I0319 11:23:50.586] # ERROR 2005 (HY000): Unknown MySQL server host 'mariadb-6646' (-2)
```

**Comment by PaulCharlton** [comment]
chart version number was not updated upon merge/rebase

**Comment by PaulCharlton** [comment]
@prydonius rebased to include 136c048bb89ad1a2039ae30919bd8a05fc19874c and bumped chart version number

**Comment by prydonius** [comment]
@PaulCharlton nice, thanks!

/lgtm

**Pull request merged**